vulcan/lib/server/controllers/vulcan_v2_controller.rb

```diff
@@ -76,7 +76,8 @@ def create_workspace
           target_mapping: @snakemake_manager.generate_target_mapping(workspace_dir, config),
           path: workspace_dir,
           user_email: @user.email,
-          git_version: @escaped_params[:git_version],
+          git_request: @escaped_params[:git_request],
```

Comment:
why do we need a `git_request` and `git_version`?

Reply:
what does an example request look like?

Reply:
`git_version` logs the sha that's actually been checked out.  `git_request` logs what the user asked for.  When `git_request` is a tag (these are editable) or a branch, this will be distinct from the `git_version`.  Andddd `git_version` could even end up being distinct for two workspaces that `git_request`ed the same branch if the workflow had had updates between when the workspaces were created.

Reply:
ah gotcha - so do we need to actually store the `git_request`? I

Reply:
doesn't have to include the word `type` but a word that describes its association with `git_revision` would be helpful.

Reply:
resolved outside of github to:
- `git_ref`: e.g. "main" / "master" / "373c1c8" for the reference string requested by the user and used for `git checkout`
- `git_sha`: e.g. "373c1c8" for the SHA of the repo ultimately checked out!